.github/workflows/ci-pytorch-tests.yml

```
@@ -194,6 +194,10 @@ jobs:
         name: unittest-results-${{ runner.os }}-py${{ matrix.python-version }}-${{ matrix.requires }}-${{ matrix.release }}
         path: tests/tests_pytorch/results-${{ runner.os }}-py${{ matrix.python-version }}-${{ matrix.requires }}-${{ matrix.release }}.xml
 
+    - name: Adjust examples
+      if: ${{ matrix.pkg-name != 'lightning' }}
```

(ignore me if I'm wrong...)
```suggestion
      if: ${{ matrix.pkg-name == 'lightning' }}
```

---

good catch :)